Readme.md

```diff
@@ -2,34 +2,66 @@
 # ipm package: sendgrid-email-library
 
 ## Overview
-SendGrid is a email service trusted by developers and marketers for time-savings, scalability and delivery expertise.  
+SendGrid is an email service trusted by developers and marketers for time-savings, scalability and delivery expertise.  
 
-To get an overview of SendGrid features and functionality click [here](). If any suggestions or addition of new methods create a pull request. For further support, comment or create issues on this repo.
+To get an overview of SendGrid features and functionality click [here](https://sendgrid.com). 
+
+If any suggestions or addition of new methods create a pull request. For further support, comment or create issues on this repo.
 
 This is an ipm package, which contains one or more reusable assets within the ipm Community. The 'package.json' in this repo is a ipm spec's package.json, [here](https://docs.clearblade.com/v/3/6-ipm/spec), which is a superset of npm's package.json spec, [here](https://docs.npmjs.com/files/package.json).
 
 [Browse ipm Packages](https://ipm.clearblade.com)
 
 ## Setup
-If you already have a SendGrid account,  API key can be generated/accessed in your account. Click [here](https://sendgrid.api-docs.io/v3.0/how-to-use-the-sendgrid-v3-api/api-authentication#api-keys) for further details. 
+
+- Create a free SendGrid Account. 
+- Log into your SendGrid account, and view the Settings > API Keys tab. Create an API Key with full access to "Mail Send" rights.
+- Update `SendGridConstants` Library
+
+```js
+// TODO Use your own credentials, these are examples
+var SEND_GRID_TOKEN = "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"
```

Comment:
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

this is invalidated, the one you have used is active.

Reply:
this are invalid on purpose